misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/command/BindgenCommand.java

```diff
@@ -38,6 +42,8 @@
 
     private static final PrintStream outStream = System.out;
     private static final PrintStream outError = System.err;
+    private Path sourceRootPath = Paths.get(System.getProperty("user.dir"));
+    private Path targetOutputPath = sourceRootPath;
```

Comment:
Is there a possibility to use one variable for `targetOutputPath` and `sourceRootPath`?

Reply:
Done.